mne/_freesurfer.py

```diff
@@ -486,6 +486,26 @@ def _ensure_image_in_surface_RAS(image, subject, subjects_dir):
     return image  # returns MGH image for header
 
 
+def _get_affine_from_lta_info(lines):
+    """Get the vox2ras affine from lta file info."""
+    # get the size of the volume (number of voxels)
+    dims = np.loadtxt([lines[0].split('=')[1]])
+    # get slice resolution
+    deltas = np.loadtxt([lines[1].split('=')[1]])
+    # get matrix of directional cosines
+    # see nibabel/freesurfer/mghformat->MGHHeader.get_affine
+    Mdc = np.loadtxt(
+        [line.split('=')[1] for line in lines[2:5]])
```

Comment:
[nitpick] this doesn't follow our variable naming conventions. The suggestion below may be a bit too verbose, can you find a middle ground? also for `MdcD`. Bonus point for a more transparent name for `cras`.

```suggestion
    matrix_of_directional_cosines = np.loadtxt(
        [line.split('=')[1] for line in lines[2:5]])
```

Reply:
I copied it from `nibabel` who copied it from freesurfer, I'll change it to follow naming conventions, I think `dir_cos` is probably fine though